package.json

```diff
@@ -8,17 +8,24 @@
   },
   "dependencies": {
     "@mapbox/polyline": "^0.2.0",
+    "metro": "^0.26.0",
```

Comment:
Is this needed? It should be required by `react-native`.

Reply:
Yes sorry, I'll remove it. I had issue with the `metro ` cache.